gradle.properties

```
@@ -1,9 +1,9 @@
 org.gradle.caching=true
 group=io.ballerina
-version=1.7.0-SNAPSHOT
+version=1.7.1-SNAPSHOT
```

Shouldn't it be 1.8.0 ? 

```suggestion
version=1.8.0-SNAPSHOT
```